src/modules/aws-ec2/termination-protection/enableTerminationProtection.ts

```diff
@@ -0,0 +1,4 @@
+import { changeTerminationProtection } from "./changeTerminationProtection";
+export const enableTerminationProtection = (instanceId: string) => {
+  changeTerminationProtection(instanceId, "true");
```

Comment:
If this is async, consider returning it so it can be properly awaited by the caller.